src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -167,7 +173,9 @@ const NavHeader = (props) => {
             )}
           </View>
         </View>
-        {!hasSearch && (
+      </View>
+      <Animated.View style={[styles.headerContainer]}>
+        {!isHeaderHide ? (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for rendering the header container has been changed from `!hasSearch` to `!isHeaderHide`. Make sure this change aligns with the intended behavior of the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->